src/components/search/result/Filter.jsx

```diff
@@ -98,6 +100,20 @@ class Filter extends Component {
     onValueCommit(history, id, value, checkbox.checked);
   }
 
+  handleCheckboxFocus(event) {
```

Comment:
Move this above and outside of the class definition, e.g.
```
const handleCheckboxFocus = (event) => {
   ...
```

Then you don't need to disable the lint rule.

Reply:
done @ray-lee. thank you for you support!